src/current/v23.2/migration-strategy-live-migration.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Migration Strategy: Live Migration"
+summary: Learn about the 'Live Migration' data migration strategy
+toc: true
+docs_area: migrate
+---
+
+This topic discusses the *live migration* strategy for [migrating your data to CockroachDB]({% link {{ page.version.version }}/migration-overview.md %}). See [Develop a migration plan]({% link {{ page.version.version }}/migration-overview.md %}#develop-a-migration-plan) for detailed information on deciding which migration strategy will work best for your application.
+
+During a live migration you maintain two production databases (the *source* and *target* database) for a period of time, and replicate data between them until a final cutover to the target database. The purpose of a live migration is to minimize downtime during this final cutover period and therefore, maintain service continuity.
+
+A live migration consists of three distinct phases:
+
+- The initial phase where the source database is the *source of truth*: the database used by the application to read and write data, and the source for replicating the data to the target database.
+- The cutover phase where CockroachDB is the source of truth and the source database is the secondary database.
+- The final phase where the source database is removed as a replication target and failover database.
+
+There are many possible approaches to performing a live migration. In this topic, we describe two example approaches that have been successful with CockroachDB: consistent cutover, and immediate cutover.
+
+The consistent cutover approach uses only one database that is writable and consistently readable at the application level, with the data replicated on the back end by another service. When using the consistent cutover approach, one database is the source of truth at any point in time. During the cutover phase, any current application transactions that use the source database are allowed to complete, and the replication service is allowed to catch up. Then new application sessions are allowed to start using the target database, and the replication service replicates data from the target database to the source database. If any problems arise during the cutover phase and you have to revert back to the initial phase, the same process repeats, moving from the target database back to the source database.
```

Comment:
I would def love a diagram here

Reply:
esp with the moving parts and the conditional logic / loop, could def imagine eyes glazing over and missing stuff here

Reply:
since we recommend this below, may be worth saying in this paragraph that this is the recommended approach